device/thunder/Cargo.toml

```diff
@@ -29,3 +29,14 @@ crate-type = ["cdylib"]
 [dependencies]
 thunder_ripple_sdk = { path = "../thunder_ripple_sdk" }
 serde = { version = "1.0", features = ["derive"] }
+pact_consumer = {version = "=0.10.4", optional = true}
+reqwest = {version = "0.11", optional = true}
+expectest = {version = "0.12.0", optional = true}
+pact-plugin-driver = {version = "0.4.1", optional = true}
+pact_models = {version = "1.0.9", optional = true}
+maplit = {version = "1.0.2", optional = true}
+test-log = {version = "0.2.11", optional = true}
+url = {version = "2.2.2", optional = true}
```

Comment:
@kevinshahfws are these not better suited to the [dev-dependencies] section of the toml as they are only used in testing and not in prod?

Reply:
Will close this PR... Now changes are in PR #170 
But to answer, yes it is purely for contract_tests feature specific